packages/core/src/styles/typography/_typography.scss

```diff
@@ -182,13 +182,18 @@
 }
 
 [cds-text~='link'] {
-  color: $cds-alias-status-info !important;
+  color: $cds-global-typography-link-color !important;
   text-decoration: underline !important;
   line-height: inherit !important;
   font-size: inherit !important;
 
   &:hover {
-    color: $cds-alias-status-info-shade !important;
+    filter: brightness(85%);
```

Comment:
Is there a reason for switching to using filter here? My concern is needing different numeric values depending on themes (dark theme or another custom theme). In the past we had encountered issue with using a percentage like this because one value did not work across different themes.

Reply:
Good point, I think I'll need to add a new token for hover states. Maybe something like this

```typescript
// tokens
link: {
    color: {
      value: token(color.blue[800]),
      hover: token(color.blue[900]),
      visited: token(color.lavender[600]),
      visitedHover: token(color.lavender[700]),
    },
  },
```

```scss
// darktheme
--cds-global-typography-link-color: #{$cds-global-color-blue-400};
--cds-global-typography-link-color-hover: #{$cds-global-color-blue-500};
--cds-global-typography-link-color-visited: #{$cds-global-color-lavender-300};
--cds-global-typography-link-color-visited-hover: #{$cds-global-color-lavender-400};
```

Reply:
Thank you! I think that would work great. I remember a case where the color contrast not being enough depending on the base color (when using a percentage). Being able to customize like this would future proof this. 👍

Reply:
Added new hover tokens for links